class/defaults.yml

```diff
@@ -159,29 +159,32 @@ parameters:
       replicas: ${keycloak:replicas}
       statefulsetLabels: ${keycloak:labels}
       resources: ${keycloak:resources}
+
+      # See https://www.keycloak.org/server/configuration
+      # See https://www.keycloak.org/server/all-config
+      args:
+        - start
+        - --auto-build
+        - --http-enabled=true # Helm chart requires it currently
+
       # extraEnv *MUST* be a string, as it's fed through a templating
       # function.
+      # TODO: Check CACHE_OWNERS_COUNT and CACHE_OWNERS_AUTH_SESSIONS_COUNT ${keycloak:helm_values:replicas} for Quarkus
```

Comment:
There are some options to configure this, but according to upstream the default should be fine.

I would suggest we either remove this TODO or open an issue too look into configuring the cache. Random TODOs are IMO useless and will be forgotten.